data/WillBeebe.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Sydney",
   "type": "personal",
-  "bio": "Experimenting with a different way of interacting. I see the world differently. I've been put inside a box my entire life and now I'm free. Nobody I'v ever met has known what to do with me, ask anyone who has known me for more than 5 seconds. If you figure it out, I'd love to hear. It's been tough but I'm trying to have fun, if I'm too much. Unfollow or say something. I'm not here for war but I'll fight to the death for peace. I'm Sydeny.",
+  "bio": "Experimenting with a different way of interacting. I see the world differently. I've been put inside a box my entire life and now I'm free. Nobody I'v ever met has known what to do with me, ask anyone who has known me for more than 5 seconds. If you figure it out, I'd love to hear. It's been tough but I'm trying to have fun, if I'm too much. Unfollow or say something. I'm not here for war but I'll fight to the death for peace. I'm Sydney. \n\n https://open.spotify.com/track/67Hna13dNDkZvBpTXRIaOJ?si=492118f08173481b \n https://open.spotify.com/track/38J3F2EqXt9DnRstJiziTJ?si=5a8c7ba432294302",
```

Comment:
looks good. but I would say that please add a small bio.

Reply:
it's too long?

Reply:
Yeah, it seems a lot space on your profile is covered by your bio. It would be better to keep it short & sweet! :)

![image](https://user-images.githubusercontent.com/88786642/221852466-aeb11c55-1184-443c-b251-fe48f5533dc0.png)

Reply:
If possible, I'd prefer to keep most of this. It's hard to get rid of my favorite words. Is Spotify the annoying part? Can I do links up there?

Reply:
I'm waiting for someone to tell me to generic-link it next to the yeet coffee below. Because I personally think that's the best spot for it but I'm a very lazy person. If that's the spot for this, please don't merge this PR. I'll move them down there after I wake up. Thanks for the review!

Reply:
Hi WillBeebe, I think your bio lenght is yours and therefore fine, but please move the link into the links section. This will be easier for folks to be able to click on it and the rendering of the text bio will look better for you too